piffle/iiif_dataclasses/__init__.py

```diff
@@ -0,0 +1,46 @@
+from __future__ import annotations
+
+from .presentation2 import (
```

Comment:
I prefer fully qualified imports, but it's possible we'll move things around so I'm ok with this for now.